.github/workflows/linters.yml

```diff
@@ -51,12 +51,10 @@ jobs:
           path: "**/dist/"
           # actions/cache will not upload changes in cache if primary key hits
           # by adding date to the primary key, we can ensure the cache updates on the first build of the day
-          key: ${{ runner.os }}-tsc-${{ hashFiles('**/tsconfig.json') }}-${{ hashFiles('**/pnpm-lock.yaml') }}-${{ steps.get-date.outputs.date }}
+          key: ${{ runner.os }}-tsc-${{ hashFiles('packages/**/tsconfig.json') }}-${{ hashFiles('pnpm-lock.yaml') }}-${{ steps.get-date.outputs.date }}
           # Allow fallback to cache of different dependencies but not allowing fallback to different tsconfig
           # because that might indicates a structural/flags changes in tsc emit.
-          restore-keys: |
-            ${{ runner.os }}-tsc-${{ hashFiles('**/tsconfig.json') }}-${{ hashFiles('**/pnpm-lock.yaml') }}
-            ${{ runner.os }}-tsc-${{ hashFiles('**/tsconfig.json') }}
+          restore-keys: ${{ runner.os }}-tsc-${{ hashFiles('packages/**/tsconfig.json') }}-${{ hashFiles('pnpm-lock.yaml') }}
```

Comment:
This change removes the ability to a fallback cache when the lock file has changed, which is tolerable in the tsc case. I'd prefer to revert this change.

Reply:
https://github.com/DimensionDev/Maskbook/runs/2891126512?check_suite_focus=true

```
hashFiles('**/tsconfig.json')
```

triggle timeout......